Modules/System Tests/Email/src/EmailTest.Codeunit.al

```diff
@@ -1206,6 +1206,70 @@ codeunit 134685 "Email Test"
         Assert.ExpectedError(EmailMessageOpenPermissionErr);
     end;
 
+    [Test]
```

Comment:
A test is needed that verifies that at least two email outbox can be retrieved and looped over

Reply:
I have changed test according to your request, but I got another issue with "RecRef" parameter. Standard tests use random values in TableId and SystemId of source records:

```
TableId := Any.IntegerInRange(1, 1000);
SystemId := Any.GuidValue();
```
Using this we will always get errors on RecRef.Open and RecRef.GetBySystemId
In last commit, I have used the first record from the Company table as a source for TableId and SystemId, but I think it is inappropriate.

I would assume to use the previous overload of GetEmailOutboxForRecord(TableId,SystemID) procedure for testing